Examples/Swift/MGLMapView+CustomAdditions.h

```diff
@@ -0,0 +1,10 @@
+#import <Mapbox/Mapbox.h>
+
+@interface MGLMapView (CustomAdditions) <CLLocationManagerDelegate>
+
+// FIXME: This will be removed once https://github.com/mapbox/mapbox-gl-native/issues/6867 is implemented
+- (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray<CLLocation *> *)locations;
+
+@property (nonatomic, readonly) CLLocationManager *locationManager;
+
+@end
```

Comment:
@frederoni is this file necessary?

Reply:
Unfortunately, it's necessary until we rewrite the custom example to use something like the new course tracking view. We could reuse this implementation but having the custom example rely on MapboxNavigation contradicts the whole purpose of the custom navigation example.

Reply:
@frederoni so if we were to merge this now, in what case is it necessary for developers to add this file?

Reply:
You only need it for simulating location updates in the custom example/MapboxCoreNavigation. Same as prior to this PR but we moved it from `MGLMapView+MGLNavigationAdditions.h` to `+CustomAdditions`.